core/src/main/java/io/kestra/core/runners/FlowInputOutput.java

```diff
@@ -345,13 +348,20 @@ public Map<String, Object> typedOutputs(
         if (flow.getOutputs() == null) {
             return ImmutableMap.of();
         }
+        final ObjectMapper mapper = new ObjectMapper();
         Map<String, Object> results = flow
             .getOutputs()
             .stream()
             .map(output -> {
-                Object current = in == null ? null : in.get(output.getId());
+                String current = in == null ? null : in.get(output.getId()).toString();
+                Object currentValue;
+                try {
+                    currentValue = mapper.readValue(current, new TypeReference<HashMap<String, Object>>() {}).get("value");
+                } catch (JsonProcessingException e) {
+                    throw new RuntimeException(e);
+                }
                 try {
-                    return parseData(execution, output, current)
+                    return parseData(execution, output, currentValue)
```

Comment:
Can you explain this part?

Reply:
As is shown in `ExecutorService`, the entryInfo now is like `{id: {value: someValue, displayName: someName}}` instead of a map with only one key like `{id: value}` before. So we need to parse this json string to a map first.

Reply:
@Skraye could you help review again?